edx_proctoring/templates/timed_exam/ready_to_submit.html

```diff
@@ -7,7 +7,12 @@
   </h3>
   <p>
     {% blocktrans %}
-      Make sure your answers are ready to be submitted, and then submit your exam. Your exam will then be graded.
+      Make sure that you have selected "Check" or "Final Check" for each problem before you submit your exam.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+      After you submit your exam, your exam will be graded.
```

Comment:
I'm not sure what the original intention of the "Your exam will then be graded" text was, but this is just a rewording of that original text. Submitting the exam causes a proctored/timed exam attempt to be recorded. Is it possible the grade wouldn't appear on the progress page until the exam is submitted? I can check that.